app/frontend/app/controllers/story.js

```diff
@@ -51,6 +59,14 @@ export default Ember.ObjectController.extend(HasCurrentUser, {
 
       var self = this;
       this.store.find('user', this.get('currentUser.id')).then(function(user) {
+        // group permission check
```

Comment:
Why even allow this event to be triggered? If I'm not a member I should  not be shown the form.